presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestCovarianceSampAggregation.java

```diff
@@ -46,27 +45,12 @@ protected String getFunctionName()
     }
 
     @Override
-    public Object getExpectedValue(int start, int length)
+    public Number getExpectedValue(int start, int length)
     {
         if (length <= 1) {
             return null;
         }
         Covariance covariance = new Covariance();
-        final double expected = covariance.covariance(constructDoublePrimitiveArray(start + 5, length), constructDoublePrimitiveArray(start, length), true);
-        return new Object() {
-            @Override
-            public boolean equals(Object obj)
-            {
-                Assert.assertEquals(expected, (Double) obj, 1e-10);
-                return true;
-            }
-
-            @Override
-            public int hashCode()
-            {
-                // required by checkstyle
-                return super.hashCode();
-            }
-        };
+        return covariance.covariance(constructDoublePrimitiveArray(start + 5, length), constructDoublePrimitiveArray(start, length), true);
```

Comment:
We don't need anonymous classes any more as our equalToForTests uses error bounds in double comparisons